test/lib/mocks.json

```diff
@@ -245,6 +278,24 @@
         "hCode": 604376696
       }
     },
+    {
```

Comment:
only this mock is needed, all the other ones can be dropped, including the xpath one.